### ISSUE TITLE
Exiv2 command-line summary reports avif image size 0x0

### ISSUE THREAD

**Issue by clanmills** [Exiv2/exiv2#1507]
Exiv2 v0.27.4 RC1 Summary Report has image size as 0x0.  The image size is correctly detected by the metadata parser as 1920,1280.

I don't consider this to be a "must fix" issue.  If "must fix" issues are detected with RC1, I will fix this for RC2.

I believe the "Summary Report" in the exiv2 command-line utility was something from the very early days of the Exiv2 project.  It reports data such as MIME type which are not in the Exif/IPTC/XMP/ICC metadata.  Life would be simpler if this were not reported!

```
643 rmills@rmillsm1:~/gnu/github/exiv2/unstable/xcode_build $ exiv2 ~/Downloads/download.avif 
File name       : /Users/rmills/Downloads/download.avif
File size       : 175706 Bytes
MIME type       : image/avif
Image size      : 0 x 0
/Users/rmills/Downloads/download.avif: No Exif data found in the file
644 rmills@rmillsm1:~/gnu/github/exiv2/unstable/xcode_build $ exiv2 -pR ~/Downloads/download.avif 
Exiv2::BmffImage::boxHandler: ftyp        0->32 brand: avif
Exiv2::BmffImage::boxHandler: meta       32->779 
  Exiv2::BmffImage::boxHandler: hdlr       44->40 
  Exiv2::BmffImage::boxHandler: pitm       84->14 
  Exiv2::BmffImage::boxHandler: iloc       98->30 
       106 |       14 |   ID |    1 |    819,174887
  Exiv2::BmffImage::boxHandler: iinf      128->40 
    Exiv2::BmffImage::boxHandler: infe      142->26 ID =   1 av01Color 
  Exiv2::BmffImage::boxHandler: iprp      168->643 
    Exiv2::BmffImage::boxHandler: ipco      176->611 
      Exiv2::BmffImage::boxHandler: ispe      184->20 pixelWidth_, pixelHeight_ = 1920, 1280
      Exiv2::BmffImage::boxHandler: pixi      204->16 
      Exiv2::BmffImage::boxHandler: av1C      220->12 
      Exiv2::BmffImage::boxHandler: colr      232->536 
      Exiv2::BmffImage::boxHandler: colr      768->19 
    Exiv2::BmffImage::boxHandler: ipma      787->24 
Exiv2::BmffImage::boxHandler: mdat      811->174895 
645 rmills@rmillsm1:~/gnu/github/exiv2/unstable/xcode_build $ 
```
[download.avif.zip](https://github.com/Exiv2/exiv2/files/6206375/download.avif.zip)

**Comment by clanmills** [comment]
```diff
index b9ad9436..5419f37a 100644
--- a/src/bmffimage.cpp
+++ b/src/bmffimage.cpp
@@ -162,7 +162,7 @@ namespace Exiv2
         if (imageWidth != exifData_.end() && imageWidth->count() > 0) {
             return imageWidth->toLong();
         }
-        return 0;
+        return pixelWidth_;
     }
 
     int BmffImage::pixelHeight() const
@@ -171,7 +171,7 @@ namespace Exiv2
         if (imageHeight != exifData_.end() && imageHeight->count() > 0) {
             return imageHeight->toLong();
         }
-        return 0;
+        return pixelHeight_;
     }
 
     std::string BmffImage::uuidName(Exiv2::DataBuf& uuid)
```

I'm not going to update the test suite concerning this at the moment.  Testing the "Summary Report" is tedious.  I'll like to see the summary report re-engineered for 0.28 to give the use for the data items to be displayed.  When that work is carried out, the summary report testing will be greatly enhanced.

**Comment by clanmills** [comment]
Fix submitted: PR #1512
